include/sw_interface/BaseControllerROS.h

```diff
@@ -108,6 +113,10 @@ class BaseControllerROS
    void cbCmdVel(const geometry_msgs::Twist::ConstPtr& cmd_vel);
    void checkAndApplyCmdVel();
 
+   bool checkFirmwareVersion(const int major_ver, const int minor_ver, const int patch_ver);
```

Comment:
Firmware version is missleading. The evo-mbed library version is checked not the firmware version of the componentes. A firmware version check is already performend in the interface nodes of the devices.

Reply:
fixed this with commit 048a2e7